kornia/augmentation/augmentation3d.py

```diff
@@ -722,6 +746,8 @@ class RandomPerspective3D(AugmentationBase3D):
                                  applied to each. Default: False.
         same_on_batch (bool): apply the same transformation across the batch. Default: False.
         align_corners(bool): interpolation flag. Default: False.
+        keepdim (bool): whether to keep the output shape the same as input (True) or broadcast it
+                        to the batch form (False). Default: False
```

Comment:
Please add an end point to all the docs.
```suggestion
        keepdim (bool): whether to keep the output shape the same as input (True) or broadcast it
                        to the batch form (False). Default: False.
```